### PR TITLE
feat: [SRM-13154]: adding more tests

### PR DESCRIPTION
**Pull request by harshilbits**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by harshilbits** [comment]
retrigger titlecheck

**Review by praneshg239 — APPROVED** [review]

File: src/modules/85-cv/components/CommonMultiItemsSideNav/tests/CommonMultiItemsSideNav.test.tsx [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+          onSelectMetric={jest.fn()}
+          isValidInput={true}
+          renamedMetric="app1"
+          openEditMetricModal={jest.fn()}
+        />
+      </TestWrapper>
+    )
+
+    await waitFor(() => expect(getByText('app1')).not.toBeNull())
+    getByText('app2')
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/CommonMultiItemsSideNav/tests/CommonMultiItemsSideNav.test.tsx:37]: are we missing assertion here ?

File: src/modules/85-cv/components/CommonMultiItemsSideNav/tests/CommonMultiItemsSideNav.test.tsx [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+    // delete second app
+    fireEvent.click(deleteButtons[1])
+    await waitFor(() => expect(container.querySelectorAll('[data-icon="main-delete"]').length).toBe(2))
+    expect(onRemoveMock).toHaveBeenCalledWith('app2', 'app1', ['app1', 'app3'], 0)
+
+    expect(container.querySelector('[class*="isSelected"]')?.innerHTML).toEqual('app1')
+  })
+
+  test('Ensure that only when app is there delete button does not exist in CommonMultiItemsSideNav', async () => {
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/CommonMultiItemsSideNav/tests/CommonMultiItemsSideNav.test.tsx:81]: guess we need to update description here 
Ensure that only when `single` app is there delete button does not exist in CommonMultiItemsSideNav

File: src/modules/85-cv/components/CommonMultiItemsSideNav/tests/CommonMultiItemsSideNav.test.tsx [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+        />
+      </TestWrapper>
+    )
+
+    await waitFor(() => expect(getByText('app1')).not.toBeNull())
+    expect(container.querySelector('[class*="isSelected"]')?.innerHTML).toEqual('app1')
+    expect(container.querySelectorAll('[data-icon="main-delete"]').length).toBe(0)
+  })
+
+  test('Ensure that when selected app nam changes, the nav shows that change in CommonMultiItemsSideNav', async () => {
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/CommonMultiItemsSideNav/tests/CommonMultiItemsSideNav.test.tsx:103]: spellcheck nam to name

Comment (harshilbits) [reply src/modules/85-cv/components/CommonMultiItemsSideNav/tests/CommonMultiItemsSideNav.test.tsx:103]: done

File: src/modules/85-cv/components/CommonMultiItemsSideNav/tests/CommonMultiItemsSideNav.test.tsx (hunk 2 above)
Comment (harshilbits) [reply src/modules/85-cv/components/CommonMultiItemsSideNav/tests/CommonMultiItemsSideNav.test.tsx:81]: done.

**Review by deepesh-ui — APPROVED** [review]

**Review by karansaraswat19 — APPROVED** [review]

**Pull request merged**